server.js

```diff
@@ -12,6 +16,9 @@ app.use(function(req, res, next) {
 
 const port = process.env.PORT || 5000;
 
+// database connection 
+connectdb()
+
 // router for all the project endpoint
 app.use("/project", projectRoute);
 // router for all the files endpoint
```

Comment:
i can't seem to find the `app.listen()` function

Reply:
It's there on the code I guess why it didn't show up was because I didn't edit it  but I would edit the code owner and create a new pull request